shm/coordinates.cpp

```diff
@@ -0,0 +1,8 @@
+#include "coordinates.hpp"
+
+Coordinates::Coordinates(const int positionX, const int positionY)
```

Comment:
const is not necessary. It is safer to use int32_t. You have to decide where is the point 0,0 of the map. If it is in the center then use int32_t, int32_t if it is in bottom left corner then uint32_t, uint32_t

Reply:
Done, tanks for comment. I decided to have 0,0 at the middle of the map